lib/cequel/record/data_set_builder.rb

```diff
@@ -99,6 +101,12 @@ def set_consistency
         end
       end
 
+      def set_allow_filtering
```

Comment:
i don't think internality is a particularly compelling excuse for omitting method docs. However, it is a small thing.